src/activities/Elsa.Activities.Reflection/Activities/ExecuteFunction.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Elsa.Attributes;
+using Elsa.Results;
+using Elsa.Services;
+using Elsa.Services.Models;
+
+namespace Elsa.Activities.Reflection.Activities
+{
+    /// <summary>
+    /// Execute a Method by reflection.
+    /// </summary>
+    [ActivityDefinition(
+        Category = "Reflection",
+        Description = "Execute a Method by reflection.",
+        RuntimeDescription = "a => !!a.state.variableName ? `Execute a Method by reflection and store the result into <strong>${ a.state.variableName }</strong>.` : 'Execute a Method by reflection.'",
+        Outcomes = new[] { OutcomeNames.Done }
+    )]
+    public class ExecuteMethod : Activity
+    {
+
+        public ExecuteMethod(IWorkflowExpressionEvaluator evaluator)
+        {
+
+        }
+
+
+        [ActivityProperty(Hint = "The variables to use as parameters, seperated by comma in order of method call.")]
+        public string InputVariableNames
+        {
+            get => GetState<string>(null, "InputVariableNames");
+            set => SetState(value, "InputVariableNames");
+        }
+
+        [ActivityProperty(Hint = "The name of the variable to store the returned value into.")]
+        public string OutputVariableName
```

Comment:
Given the fact that this activity represents a function to execute, I think it's only appropriate to have this activity store the function result into the activity's Output, rather than letting the user specify a variable name to store it into. The result value can then be accessed using an expression, e.g. `{{ Activities.MyMethodActivity.Result }}`

Reply:
Let ne Take a look in that.